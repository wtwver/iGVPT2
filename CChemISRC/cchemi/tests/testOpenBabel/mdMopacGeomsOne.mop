PM7 GNORM=0.001 XYZ CHARGE=1 Singlet

Mopac file generated by Gabedit
 C 1.113516 1 0.608063 1 0.711227 1
 O -0.069650 1 1.229830 1 0.187232 1
 C -1.235660 1 0.709772 1 0.769490 1
 O -2.244564 1 1.612527 1 0.412485 1
 C -1.546358 1 -0.636679 1 0.060042 1
 N -2.226577 1 -0.353660 1 -1.271715 1
 C -0.201502 1 -1.398515 1 -0.242665 1
 O 0.093722 1 -1.187447 1 -1.588634 1
 C 0.938187 1 -0.920347 1 0.709266 1
 O 0.434281 1 -1.275309 1 1.981950 1
 C 2.200737 1 1.123941 1 -0.233819 1
 O 2.422199 1 0.122085 1 -1.213426 1
 H 1.223923 1 1.019543 1 1.729554 1
 H -1.170827 1 0.630424 1 1.865276 1
 H -1.917179 1 2.562514 1 0.462364 1
 H -2.196232 1 -1.249349 1 0.711015 1
 H -2.474369 1 -1.245564 1 -1.705663 1
 H -0.342689 1 -2.487533 1 -0.161791 1
 H 0.911722 1 -0.610721 1 -1.690010 1
 H 1.878951 1 -1.442932 1 0.461439 1
 H 1.112283 1 -1.778638 1 2.496116 1
 H 3.146290 1 1.318611 1 0.308089 1
 H 1.894428 1 2.079228 1 -0.712319 1
 H 3.068274 1 0.430964 1 -1.891312 1
 H -1.549286 1 0.132169 1 -1.903247 1
 H -3.069726 1 0.239379 1 -1.139142 1
