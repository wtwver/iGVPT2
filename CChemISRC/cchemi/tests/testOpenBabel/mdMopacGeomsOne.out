 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 158    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Sat Dec 19 19:36:03 2015  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     1
 *
 *
 *
 *  XYZ        - CARTESIAN COORDINATE SYSTEM TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GNORM=     - EXIT WHEN GRADIENT NORM DROPS BELOW  0.100E-02
 *******************************************************************************
PM7 GNORM=0.001 XYZ CHARGE=1 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          1.11351600  *   0.60806300  *   0.71122700  *
     2       O         -0.06965000  *   1.22983000  *   0.18723200  *
     3       C         -1.23566000  *   0.70977200  *   0.76949000  *
     4       O         -2.24456400  *   1.61252700  *   0.41248500  *
     5       C         -1.54635800  *  -0.63667900  *   0.06004200  *
     6       N         -2.22657700  *  -0.35366000  *  -1.27171500  *
     7       C         -0.20150200  *  -1.39851500  *  -0.24266500  *
     8       O          0.09372200  *  -1.18744700  *  -1.58863400  *
     9       C          0.93818700  *  -0.92034700  *   0.70926600  *
    10       O          0.43428100  *  -1.27530900  *   1.98195000  *
    11       C          2.20073700  *   1.12394100  *  -0.23381900  *
    12       O          2.42219900  *   0.12208500  *  -1.21342600  *
    13       H          1.22392300  *   1.01954300  *   1.72955400  *
    14       H         -1.17082700  *   0.63042400  *   1.86527600  *
    15       H         -1.91717900  *   2.56251400  *   0.46236400  *
    16       H         -2.19623200  *  -1.24934900  *   0.71101500  *
    17       H         -2.47436900  *  -1.24556400  *  -1.70566300  *
    18       H         -0.34268900  *  -2.48753300  *  -0.16179100  *
    19       H          0.91172200  *  -0.61072100  *  -1.69001000  *
    20       H          1.87895100  *  -1.44293200  *   0.46143900  *
    21       H          1.11228300  *  -1.77863800  *   2.49611600  *
    22       H          3.14629000  *   1.31861100  *   0.30808900  *
    23       H          1.89442800  *   2.07922800  *  -0.71231900  *
    24       H          3.06827400  *   0.43096400  *  -1.89131200  *
    25       H         -1.54928600  *   0.13216900  *  -1.90324700  *
    26       H         -3.06972600  *   0.23937900  *  -1.13914200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        1.1135    0.6081    0.7112
     2         O       -0.0697    1.2298    0.1872
     3         C       -1.2357    0.7098    0.7695
     4         O       -2.2446    1.6125    0.4125
     5         C       -1.5464   -0.6367    0.0600
     6         N       -2.2266   -0.3537   -1.2717
     7         C       -0.2015   -1.3985   -0.2427
     8         O        0.0937   -1.1874   -1.5886
     9         C        0.9382   -0.9203    0.7093
    10         O        0.4343   -1.2753    1.9819
    11         C        2.2007    1.1239   -0.2338
    12         O        2.4222    0.1221   -1.2134
    13         H        1.2239    1.0195    1.7296
    14         H       -1.1708    0.6304    1.8653
    15         H       -1.9172    2.5625    0.4624
    16         H       -2.1962   -1.2493    0.7110
    17         H       -2.4744   -1.2456   -1.7057
    18         H       -0.3427   -2.4875   -0.1618
    19         H        0.9117   -0.6107   -1.6900
    20         H        1.8790   -1.4429    0.4614
    21         H        1.1123   -1.7786    2.4961
    22         H        3.1463    1.3186    0.3081
    23         H        1.8944    2.0792   -0.7123
    24         H        3.0683    0.4310   -1.8913
    25         H       -1.5493    0.1322   -1.9032
    26         H       -3.0697    0.2394   -1.1391

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads = 16                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C6 H14 N O5  =    26 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   36

      Geometry optimization using EF

  GNORM HAS BEEN SET TOO LOW, RESET TO 0.01

  SPECIFY LET AS KEYWORD TO ALLOW GNORM LESS THAN 0.01

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.047 TIME LEFT:  2.00D  GRAD.:    42.151 HEAT: -86.49529
 CYCLE:     2 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    29.787 HEAT: -87.15751
 CYCLE:     3 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    21.859 HEAT: -87.57211
 CYCLE:     4 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    21.026 HEAT: -87.72539
 CYCLE:     5 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    16.798 HEAT: -87.88206
 CYCLE:     6 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    22.897 HEAT: -87.88641
 CYCLE:     7 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    19.478 HEAT: -87.99170
 CYCLE:     8 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     8.727 HEAT: -88.12611
 CYCLE:     9 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     5.789 HEAT: -88.16578
 CYCLE:    10 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     9.401 HEAT: -88.17298
 CYCLE:    11 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     6.350 HEAT: -88.22016
 CYCLE:    12 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     7.068 HEAT: -88.22486
 CYCLE:    13 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.157 HEAT: -88.25247
 CYCLE:    14 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     5.789 HEAT: -88.25340
 CYCLE:    15 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     4.015 HEAT: -88.26507
 CYCLE:    16 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.667 HEAT: -88.26918
 CYCLE:    17 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.023 HEAT: -88.27364
 CYCLE:    18 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.904 HEAT: -88.27635
 CYCLE:    19 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.109 HEAT: -88.28039
 CYCLE:    20 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     6.123 HEAT: -88.27214
 CYCLE:    21 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.853 HEAT: -88.27859
 CYCLE:    22 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:     2.539 HEAT: -88.28668
 CYCLE:    23 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.491 HEAT: -88.28350
 CYCLE:    24 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.298 HEAT: -88.28481
 CYCLE:    25 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.342 HEAT: -88.29100
 CYCLE:    26 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     5.354 HEAT: -88.28247
 CYCLE:    27 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.873 HEAT: -88.29328
 CYCLE:    28 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     2.831 HEAT: -88.29233
 CYCLE:    29 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.454 HEAT: -88.29414
 CYCLE:    30 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.866 HEAT: -88.29168
 CYCLE:    31 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.524 HEAT: -88.29378
 CYCLE:    32 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     1.272 HEAT: -88.29881
 CYCLE:    33 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.564 HEAT: -88.29836
 CYCLE:    34 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.498 HEAT: -88.30066
 CYCLE:    35 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.777 HEAT: -88.30179
 CYCLE:    36 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.518 HEAT: -88.30461
 CYCLE:    37 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.130 HEAT: -88.30207
 CYCLE:    38 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.002 HEAT: -88.30922
 CYCLE:    39 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.346 HEAT: -88.30447
 CYCLE:    40 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.169 HEAT: -88.31149
 CYCLE:    41 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.150 HEAT: -88.30976
 CYCLE:    42 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.524 HEAT: -88.31170
 CYCLE:    43 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.113 HEAT: -88.30712
 CYCLE:    44 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.670 HEAT: -88.31209
 CYCLE:    45 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.075 HEAT: -88.30558
 CYCLE:    46 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.294 HEAT: -88.31556
 CYCLE:    47 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.989 HEAT: -88.31003
 CYCLE:    48 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.729 HEAT: -88.31657
 CYCLE:    49 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     3.338 HEAT: -88.31266
 CYCLE:    50 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.373 HEAT: -88.31728
 CYCLE:    51 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.555 HEAT: -88.31766
 CYCLE:    52 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.104 HEAT: -88.31834
 CYCLE:    53 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.123 HEAT: -88.32166
 CYCLE:    54 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.759 HEAT: -88.32293
 CYCLE:    55 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.722 HEAT: -88.32375
 CYCLE:    56 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.502 HEAT: -88.32450
 CYCLE:    57 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.720 HEAT: -88.32511
 CYCLE:    58 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.173 HEAT: -88.32567
 CYCLE:    59 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     0.141 HEAT: -88.32578
 CYCLE:    60 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.138 HEAT: -88.32600
 CYCLE:    61 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.102 HEAT: -88.32610
 CYCLE:    62 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     0.088 HEAT: -88.32610
 CYCLE:    63 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.066 HEAT: -88.32611
 CYCLE:    64 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.106 HEAT: -88.32614
 CYCLE:    65 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.097 HEAT: -88.32616
 CYCLE:    66 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.120 HEAT: -88.32621
 CYCLE:    67 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.166 HEAT: -88.32625
 CYCLE:    68 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.148 HEAT: -88.32635
 CYCLE:    69 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     0.158 HEAT: -88.32646
 CYCLE:    70 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.132 HEAT: -88.32652
 CYCLE:    71 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.257 HEAT: -88.32644
 CYCLE:    72 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.106 HEAT: -88.32664
 CYCLE:    73 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.122 HEAT: -88.32667
 CYCLE:    74 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.106 HEAT: -88.32668
 CYCLE:    75 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.093 HEAT: -88.32670
 CYCLE:    76 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.109 HEAT: -88.32674
 CYCLE:    77 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.149 HEAT: -88.32680
 CYCLE:    78 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.105 HEAT: -88.32690
 CYCLE:    79 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     0.080 HEAT: -88.32693
 CYCLE:    80 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.089 HEAT: -88.32692
 CYCLE:    81 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.095 HEAT: -88.32692
 CYCLE:    82 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.082 HEAT: -88.32693
 CYCLE:    83 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.109 HEAT: -88.32701
 CYCLE:    84 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.160 HEAT: -88.32715
 CYCLE:    85 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.164 HEAT: -88.32728
 CYCLE:    86 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.310 HEAT: -88.32765
 CYCLE:    87 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.154 HEAT: -88.32788
 CYCLE:    88 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.108 HEAT: -88.32791
 CYCLE:    89 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.136 HEAT: -88.32821
 CYCLE:    90 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     1.044 HEAT: -88.32905
 CYCLE:    91 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.277 HEAT: -88.32968
 CYCLE:    92 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.353 HEAT: -88.33078
 CYCLE:    93 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.015 HEAT: -88.33284
 CYCLE:    94 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.059 HEAT: -88.33394
 CYCLE:    95 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.922 HEAT: -88.33748
 CYCLE:    96 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.460 HEAT: -88.33839
 CYCLE:    97 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.285 HEAT: -88.33874
 CYCLE:    98 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.337 HEAT: -88.33855
 CYCLE:    99 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.247 HEAT: -88.33908
 CYCLE:   100 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.363 HEAT: -88.33942
 CYCLE:   101 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.210 HEAT: -88.33977
 CYCLE:   102 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.293 HEAT: -88.33970
 CYCLE:   103 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.120 HEAT: -88.33999
 CYCLE:   104 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.155 HEAT: -88.33966
 CYCLE:   105 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     0.105 HEAT: -88.34013
 CYCLE:   106 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.118 HEAT: -88.34020
 CYCLE:   107 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.193 HEAT: -88.34018
 CYCLE:   108 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.155 HEAT: -88.34026
 CYCLE:   109 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.220 HEAT: -88.34060
 CYCLE:   110 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.232 HEAT: -88.34080
 CYCLE:   111 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.315 HEAT: -88.34120
 CYCLE:   112 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.241 HEAT: -88.34127
 CYCLE:   113 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.101 HEAT: -88.34131
 CYCLE:   114 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.079 HEAT: -88.34126
 CYCLE:   115 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.054 HEAT: -88.34125
 CYCLE:   116 TIME:   0.070 TIME LEFT:  2.00D  GRAD.:     2.599 HEAT: -88.33861
 CYCLE:   117 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.926 HEAT: -88.33401
 CYCLE:   118 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.112 HEAT: -88.33354
 CYCLE:   119 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.965 HEAT: -88.33995
 CYCLE:   120 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     1.807 HEAT: -88.33826
 CYCLE:   121 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.205 HEAT: -88.34105
 CYCLE:   122 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.243 HEAT: -88.34100
 CYCLE:   123 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.041 HEAT: -88.34114
 CYCLE:   124 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.087 HEAT: -88.34121


 HEAT OF FORMATION IS ESSENTIALLY STATIONARY

 -------------------------------------------------------------------------------
PM7 GNORM=0.001 XYZ CHARGE=1 Singlet

 Mopac file generated by Gabedit


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2012 (Version: 15.146L)
                                                       Sat Dec 19 19:36:05 2015
                                                       No. of days left = 158




          FINAL HEAT OF FORMATION =        -88.34121 KCAL/MOL =    -369.61961 KJ/MOL


          TOTAL ENERGY            =      -2582.83378 EV
          ELECTRONIC ENERGY       =     -14772.15773 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =      12189.32395 EV
          COSMO AREA              =        184.68 SQUARE ANGSTROMS
          COSMO VOLUME            =        195.94 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.08707
          IONIZATION POTENTIAL    =         14.219105 EV
          HOMO LUMO ENERGIES (EV) =        -14.219 -3.311
          NO. OF FILLED LEVELS    =         36
          MOLECULAR WEIGHT        =        180.180

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    16    H    24     6.07806
            H    26    H    21     5.72047
            H    15    H    18     4.69385


          SCF CALCULATIONS        =        144
 
          WALL-CLOCK TIME         =          2.062 SECONDS
          COMPUTATION TIME        =         31.479 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          1.11246594  *   0.61823635  *   0.68944385  *
     2       O         -0.07606200  *   1.26393321  *   0.23769569  *
     3       C         -1.24675443  *   0.69934703  *   0.78814801  *
     4       O         -2.29489458  *   1.55259232  *   0.43181051  *
     5       C         -1.53923211  *  -0.62950117  *   0.03557928  *
     6       N         -2.13743419  *  -0.27889361  *  -1.31754323  *
     7       C         -0.21128283  *  -1.41436223  *  -0.20591605  *
     8       O          0.09911549  *  -1.26004386  *  -1.56110753  *
     9       C          0.92469391  *  -0.91536043  *   0.71873431  *
    10       O          0.46541009  *  -1.20662391  *   2.02181148  *
    11       C          2.19921280  *   1.11699623  *  -0.26986942  *
    12       O          2.37141481  *   0.11761226  *  -1.26182205  *
    13       H          1.30818860  *   0.99133380  *   1.72968509  *
    14       H         -1.18149813  *   0.58828957  *   1.89403103  *
    15       H         -2.07235171  *   2.50223122  *   0.60353101  *
    16       H         -2.26537147  *  -1.24944474  *   0.61881963  *
    17       H         -2.29015113  *  -1.13045577  *  -1.88095349  *
    18       H         -0.37551544  *  -2.51139038  *  -0.06475680  *
    19       H          0.94687722  *  -0.73564209  *  -1.67792036  *
    20       H          1.87844030  *  -1.44479817  *   0.48768502  *
    21       H          1.06155537  *  -1.82667512  *   2.49687534  *
    22       H          3.17061234  *   1.24669051  *   0.24985801  *
    23       H          1.89673874  *   2.06957838  *  -0.75237620  *
    24       H          2.96837733  *   0.42336925  *  -1.97972262  *
    25       H         -1.45582507  *   0.32368629  *  -1.82261242  *
    26       H         -3.02099967  *   0.24648376  *  -1.19451768  *

                             CARTESIAN COORDINATES

   1    C        1.112465938     0.618236350     0.689443854
   2    O       -0.076061999     1.263933209     0.237695687
   3    C       -1.246754425     0.699347033     0.788148013
   4    O       -2.294894575     1.552592316     0.431810508
   5    C       -1.539232114    -0.629501168     0.035579278
   6    N       -2.137434190    -0.278893610    -1.317543227
   7    C       -0.211282830    -1.414362234    -0.205916051
   8    O        0.099115486    -1.260043859    -1.561107533
   9    C        0.924693908    -0.915360430     0.718734312
  10    O        0.465410092    -1.206623907     2.021811477
  11    C        2.199212796     1.116996230    -0.269869419
  12    O        2.371414814     0.117612258    -1.261822049
  13    H        1.308188601     0.991333802     1.729685086
  14    H       -1.181498131     0.588289567     1.894031035
  15    H       -2.072351706     2.502231224     0.603531010
  16    H       -2.265371472    -1.249444743     0.618819625
  17    H       -2.290151129    -1.130455773    -1.880953490
  18    H       -0.375515444    -2.511390380    -0.064756802
  19    H        0.946877218    -0.735642093    -1.677920363
  20    H        1.878440305    -1.444798169     0.487685024
  21    H        1.061555369    -1.826675122     2.496875342
  22    H        3.170612344     1.246690507     0.249858008
  23    H        1.896738742     2.069578379    -0.752376202
  24    H        2.968377330     0.423369249    -1.979722623
  25    H       -1.455825071     0.323686290    -1.822612422
  26    H       -3.020999668     0.246483764    -1.194517680


           Empirical Formula: C6 H14 N O5  =    26 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -43.07074 -35.20720 -33.95448 -33.22435 -33.01978 -32.81851 -30.63533 -30.19680
 -27.75098 -26.89804 -24.93482 -23.83136 -23.38235 -22.85712 -22.35242 -21.98860
 -21.63851 -21.05834 -20.51049 -20.14265 -20.01504 -18.79233 -18.65495 -18.35692
 -18.18806 -17.85348 -17.41925 -16.89073 -16.83476 -16.21865 -15.99665 -15.54900
 -15.03681 -14.76380 -14.69351 -14.21911  -3.31086  -2.14040  -1.82026  -1.61669
  -1.45068  -1.28256  -1.05003  -0.73935  -0.66870  -0.57608  -0.19869  -0.15212
   0.35884   0.52620   0.61969   0.81569   1.33552   1.43098   1.50723   1.64604
   1.77459   1.83295   2.01371   2.05524   2.23766   2.62260


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.067269        3.9327     1.11026     2.82247
    2          O          -0.484109        6.4841     1.87673     4.60737
    3          C           0.359394        3.6406     1.11847     2.52214
    4          O          -0.601760        6.6018     1.84216     4.75960
    5          C          -0.143226        4.1432     1.12267     3.02056
    6          N          -0.187547        5.1875     1.47433     3.71322
    7          C           0.122515        3.8775     1.10147     2.77601
    8          O          -0.606789        6.6068     1.84998     4.75681
    9          C           0.054161        3.9458     1.10665     2.83919
   10          O          -0.571331        6.5713     1.84721     4.72412
   11          C          -0.010786        4.0108     1.09547     2.91532
   12          O          -0.607257        6.6073     1.83683     4.77043
   13          H           0.172235        0.8278     0.82776
   14          H           0.171133        0.8289     0.82887
   15          H           0.383019        0.6170     0.61698
   16          H           0.195940        0.8041     0.80406
   17          H           0.301388        0.6986     0.69861
   18          H           0.171767        0.8282     0.82823
   19          H           0.389019        0.6110     0.61098
   20          H           0.174548        0.8255     0.82545
   21          H           0.353721        0.6463     0.64628
   22          H           0.156228        0.8438     0.84377
   23          H           0.156394        0.8436     0.84361
   24          H           0.362910        0.6371     0.63709
   25          H           0.311861        0.6881     0.68814
   26          H           0.309307        0.6907     0.69069
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -2.335    -0.324    -1.186     2.639
 HYBRID        -0.130     0.584    -1.148     1.295
 SUM           -2.466     0.259    -2.334     3.405


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.11026   0.81447   0.94500   1.06299
    2  O   1.87673   1.24914   1.61654   1.74169
    3  C   1.11847   0.71657   0.79691   1.00866
    4  O   1.84216   1.48179   1.37277   1.90504
    5  C   1.12267   1.07641   1.09179   0.85236
    6  N   1.47433   1.25040   1.25353   1.20929
    7  C   1.10147   0.91628   1.06641   0.79332
    8  O   1.84998   1.60986   1.83254   1.31441
    9  C   1.10665   1.04320   0.99795   0.79803
   10  O   1.84721   1.61402   1.74978   1.36033
   11  C   1.09547   1.05885   0.94566   0.91081
   12  O   1.83683   1.76947   1.51642   1.48454
   13  H   0.82776
   14  H   0.82887
   15  H   0.61698
   16  H   0.80406
   17  H   0.69861
   18  H   0.82823
   19  H   0.61098
   20  H   0.82545
   21  H   0.64628
   22  H   0.84377
   23  H   0.84361
   24  H   0.63709
   25  H   0.68814
   26  H   0.69069

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             2.08 SECONDS

 == MOPAC DONE ==
